R/bundle.R

```diff
@@ -509,29 +502,22 @@ createAppManifest <- function(appDir, appMode, contentCategory, hasParameters,
       !identical(appMode, "tensorflow-saved-model")) {
 
     # detect dependencies including inferred dependences
-    python <- getPython(python)
     deps = snapshotDependencies(appDir, inferDependencies(appMode, hasParameters, python))
 
     # construct package list from dependencies
     for (i in seq.int(nrow(deps))) {
       name <- deps[i, "Package"]
 
-      if (name == "reticulate") {
-        if (python == "") {
-          # TODO, should this be a warning for backward compatibility?
-          msg <- c(msg, "reticulate is in use, but python was not specified")
+      if (name == "reticulate" && !is.null(python)) {
```

Comment:
Realize you're not changing this code (so no need to change it in this PR), but we typically prefer `identical(x, "y")` for checking for string equality, since it can tolerate `x` being NULL, non-strings, etc.